### PR TITLE
Prepare AppIntro 6.1.0

### PR DESCRIPTION
**Pull request by cortinico**
Time to craft a new release 🥳

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,74 @@
...
+## Version 6.1.0 *(2021-01-29)*
+
+This is a new minor release of AppIntro. This library comes with several new features (see below) and several bugfixes.
+
+### Summary of Changes
+
+* Target SDK is now 30.
+* Text visualization has been improved with Autosizing TextViews and URL autolinking
+* AppIntro now offers better support for tablets (`sw600dp`)
+* Slide indicator has been improved with be
```
Comment (paolorotolo) [inline CHANGELOG.md:12]: Typo here

**Review by paolorotolo — CHANGES_REQUESTED** [review]
Hey @cortinico, great work! Just fix the small typo and it's good to go for me 👍🏻

File: CHANGELOG.md (hunk 1 above)
Comment (cortinico) [reply CHANGELOG.md]: Nice catch 👍 fixed it

**Review by paolorotolo — APPROVED** [review]
Perfect, thanks!

**Pull request merged**